modules/opportunities/client/directives/opportunities.eval.directive.js

```diff
@@ -45,13 +45,14 @@
 				 * Weighting for opportunity evaluations
 				 * TODO: Have these pulled from the opportunity itself rather than hardcoded in
 				 */
-				vm.weights = {
-					price			: 0.1,
-					interview		: 0.25,
-					question		: 0.25,
-					skill			: 0.05,
-					codechallenge	: 0.35
-				};
+				// vm.weights = {
```

Comment:
@sutherlanda - Do we need to keep this section (or the TODO at line 46)?

Reply:
No.  I've just added another commit to remove the old comments.  I'll merge into dev now.